cli/commands/play.go

```diff
@@ -109,26 +122,50 @@ var run = func(cmd *cobra.Command, args []string) {
 		if TurnDelay > 0 {
 			time.Sleep(time.Duration(TurnDelay) * time.Millisecond)
 		}
+
+		if exportGame {
+			// The output file was designed in a way so that (nearly) every entry is equivalent to a valid API request.
+			// This is meant to help unlock further development of tools such as replaying a saved game by simply copying each line and sending it as a POST request.
+			// There was a design choice to be made here: the difference between SnakeResponse and BoardState is the `you` key.
+			// We could choose to either store the SnakeResponse of each snake OR to omit the `you` key OR fill the `you` key with one of the snakes
+			// In all cases the API request is technically non-compliant with how the actual API response should be.
+			// The third option (filling the `you` key with an arbitrary snake) is the closest to the actual API response that would need the least manipulation to
+			// be adjusted to look like an API call for a specific snake in the game.
```

Comment:
Thanks for adding this comment!
Minor typo - it looks like this refers to "responses" and "SnakeResponses", when it should be "requests" and "SnakeRequests".

Reply:
Yep, you're correct :+1:  Adjusted.